manual/python/index.md

```diff
@@ -1,70 +1,95 @@
 
-## Running PyROOT from the Python interpreter
+## Building and installing
+PyROOT is enabled by default when building and installing ROOT. Please refer to [Building ROOT from source]({{ '/install/build_from_source' | relative_url }}) for specific information on how PyROOT can be built, and to [Installing ROOT]({{ '/install' | relative_url }}) to learn about the ways to install ROOT.
 
-Ensure that the Python command is executed where the `libPyROOT.so` ROOT extension module is located. This is the main entry point for any Python script using the ROOT classes.
+## Getting started
 
-PyROOT scripts work as usual Python scripts. You just need to import ROOT:
+Once ROOT has been installed, PyROOT can be used both from the Python prompt and from a Python script. The entry point to PyROOT is the `ROOT` module, which needs to be imported first: 
 
-```
+```python
 import ROOT
 ```
 
-_**Example**_
+After that, all the ROOT C++ functionality (classes, functions, etc.) can be accessed via the `ROOT` module. In the example below, we show how to create a histogram (an instance of the `TH1F` class) and randomly fill it with a gaussian distribution.
 
+```python
+h = ROOT.TH1F("myHist", "myTitle", 64, -4, 4)
+h.FillRandom("gaus")
 ```
-from ROOT import TCanvas, TF1
 
-c1 = TCanvas( 'c1', 'Example with Formula', 200, 10, 700, 500 )
+### Tutorials
+
+There are a number of tutorials that show how to use the various ROOT features from Python. They can be found in the link below. 
+
+{% include tutorials name="PyROOT" url="pyroot" %}
+
+## Interactive graphics
+
+Just like from C++, it is also possible to create interactive ROOT graphics from Python thanks to PyROOT. As an example, let's consider the following code, which creates a one-dimensional function and draws it:
 
-# Create a one dimensional function and draw it.
-fun1 = TF1( 'fun1', 'abs(sin(x)/x)', 0, 10 )
-c1.SetGrid()
-fun1.Draw()
+```python
+>>> import ROOT
+>>> f = ROOT.TF1("f1", "sin(x)/x", 0., 10.)
+>>> f.Draw()
 ```
 
+As a result of running the code above from the Python prompt, a new canvas will open up to show the drawn function:
+
+{% include figure_image
+img="tf1_draw.png"
+caption="Example of graphics generated with PyROOT."
+%}
 
-## Running Python from the ROOT/Cling interpreter
+> **Note that** the code above can also be written in a script file and executed with Python. In that case, the script will run to completion and the Python process will terminate, making the created canvas disappear. If you want to keep the Python process alive and thus be able to inspect your canvas, run the script with `python -i script_name.py`.
 
-In ROOT can run any Python command via the {% include ref class="TPython" %} class.
+## Interface
 
- _**Example**_
+Coming soon!
 
-{% highlight C++ %}
-root [0] TPython::Exec( "print 1 + 1" )
+## Loading user libraries & jitting
+
+Coming soon!
```

Comment:
Same here, make clear we are just using the content on the webiste, not the actual feature.